src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
```

Comment:
[Not critical]
It's better to name this type something like `ErrorProps`, so that the name describes this type the most precisely.